rmltraintfbbox/rmltraintfbbox/utils/pipeline_templates/center_net_resnet101_v1_fpn_adam.config

```diff
@@ -0,0 +1,130 @@
+model {
+  center_net {
+    num_classes: <replace_num_classes>
+    feature_extractor {
+      type: "resnet_v2_101"
+    }
+    image_resizer {
+      keep_aspect_ratio_resizer {
+        min_dimension: 224
+        max_dimension: 224
+        pad_to_max_dimension: true
+      }
+    }
+    object_detection_task {
+      task_loss_weight: 1.0
+      offset_loss_weight: 1.0
+      scale_loss_weight: 0.1
+      localization_loss {
+        l1_localization_loss {
+        }
+      }
+    }
+    object_center_params {
+      object_center_loss_weight: 1.0
+      min_box_overlap_iou: 0.7
+      max_box_predictions: <replace_max_box_predictions>
+      classification_loss {
+        penalty_reduced_logistic_focal_loss {
+          alpha: 2.0
+          beta: 4.0
+        }
+      }
+    }
+  }
+}
+
+train_config: {
+
+  fine_tune_checkpoint_version: V2
+  fine_tune_checkpoint: "<replace_arch_path>checkpoint/ckpt-0"
+  fine_tune_checkpoint_type: "classification"
+
+  batch_size: <replace_batch_size>
+  num_steps: <replace_train_steps>
+
+  data_augmentation_options {
+    random_horizontal_flip {
+    }
+  }
```

Comment:
I like to add `random_vertical_flip` and `random_rotation90` to all the pipeline configs as well, since our data is orientation-independent and more augmentations can never hurt

Reply:
Added, thanks for suggestion